### DISCUSSION TITLE
Saving Views with Database folder

### DISCUSSION THREAD

**Discussion by the1gofer** [RafaelGB/obsidian-db-folder#678] [Q&A]
Is it possible to save a particular view with filters and sorting, etc for a view using DB folder view?

For example,  I frequently end up filtering by a particular category, and it would be nice to have that view save in stead of having to redo it each time.

**Comment by RafaelGB** [comment]
Hello @the1gofer. The filters are saved on disk, sure

Could you share some image?

https://user-images.githubusercontent.com/11924043/208726358-67e9c9bf-9de5-4e88-97b0-432f61ea220b.mov

**Reply by the1gofer** [reply]
mine doesn't look like yours ....

<img width="1440" alt="Screenshot 2022-12-21 at 9 36 17 AM" src="https://user-images.githubusercontent.com/2140724/208957203-8c326268-bfe3-485b-b85d-9368f142767f.png">

**Reply by RafaelGB** [reply]
![208957203-8c326268-bfe3-485b-b85d-9368f142767f](https://user-images.githubusercontent.com/11924043/208960919-37403d4e-73fd-4413-8867-4b17c5372b59.png)

Just click that icon and you will see the menu of the video
